doc/running-docker.md

```diff
@@ -1,22 +1,21 @@
+# Running the PrairieLearn docker image
```

Comment:
capital-D Docker for consistency. Maybe better as "Running PrairieLearn with Docker"

Maybe we should repeat the text here about this being the recommended way to run PL?